### PR TITLE
JuliaCon 2023 Presentation

### PR DESCRIPTION
**Pull request by mkitti**
This is a working copy of the presentation.

I'm using https://marp.app/ to generate slides from markdown

### CODE REVIEW

**Comment by mkitti** [comment]
Preview working copy: https://github.com/JuliaIO/HDF5.jl/blob/juliacon_2023_presentation/docs/juliacon_2023/juliacon_2023_presentation.pdf

**Comment by simonbyrne** [comment]
Thank you! I was able to get it set up, and made a few minor tweaks.

I'll work on the MPI bits on Sunday.

**Comment by musm** [comment]
Do you think there is time to live demo some of the code e.g. in jupyter or is the plan to walk through the slides?

**Comment by mkitti** [comment]
I think we may

**Comment by mkitti** [comment]
I think we may be able to create a notebook for people run a demo, but I think we're running out of a time for a live demo.

Simon also suggested we discuss some applications.

File: docs/juliacon_2023/juliacon_2023_presentation.md [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+---
+
+# HDF5.jl Early and Recent Contributors
+
+* There are many contributors
+* Konrad Hisen initiated Julia's support for HDF5
+* Tim Holy and Simon Kornblith were the initial primary authors
+* Tom Short, Blake Johnson, Isaih Norton, Elliot Saba, Steven Johnson, Mike Nolta, Jameson Nash
+* Justin Willmert improved many aspects C to Julia API interface
+* Other recent contributors: t-bltg, Henrik Ranocha, Nathan Zimmerberg, Joshua Lampert, Tamas Gal, David MacMahon, Juan Ignacio Polanco, Michale Schlottke-Lakemper, linwaytin, Dmitri Iouchtchenko, Lorenzo Van Munoz, Jared Wahlstrand, Julian Samaroo, machakann, James Hester, Ralph Kube, Kristoffer Carlsson
```
Comment (JoshuaLampert) [inline docs/juliacon_2023/juliacon_2023_presentation.md:81]: Two small typos: It's **Hendrik** Ranocha and **Michael** Schlottke-Lakemper :smile:

**Comment by mkitti** [comment]
Do we want to merge this somehow to be part of the documentation?

**Comment by musm** [comment]
I agree, we should merge this and link it in the docs under additional resources

**Pull request merged**